index2.js

```diff
@@ -0,0 +1,7 @@
+//У прямоугольного треугольника две стороны n (со значением 3) и m (со значением 4). Найдите гипотенузу k по теореме.
+
+const n = 3;
+const m = 4;
+const k = Math.sqrt(n**2 + m**2);
```

Comment:
Check the task attentively. You have to write a function, not a result into variable (but your logic is right)